packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.1.1
+
+- Adds missed poping log.
```

Comment:
nit: poping -> popping, here and everywhere in this pr

Reply:
Ups. Fixed by rebase.